### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by manojneuro**
@ltetrel Hardcoding the data path for neurolibre binder test. This will allow users to run the tutorials with no changes.

### CODE REVIEW

File: binder/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-brainiak
+git+https://github.com/brainiak/brainiak
```
Comment (ltetrel) [inline binder/requirements.txt:1]: did you tested the binder with this new environment ?

**Comment by manojneuro** [comment]
@ltetrel: please let me know what your comments are. From your earlier message, I don't see your comments here. This PR will make the user experience seamless.

**Comment by ltetrel** [comment]
Sorry it was pending, I fought it was sent

File: binder/requirements.txt (hunk 1 above)
Comment (manojneuro) [reply binder/requirements.txt:1]: @ltetrel I'm unfamiliar with binder builds. Is there a simple way to execute this? I thought only you can do this on your cluster. We do build brainiak using this form on google colab.

Comment (ltetrel) [reply binder/requirements.txt:1]: Ok I will just accept it, I don't think it should change anything

**Pull request merged**